src/styles/utils/_vars.scss

```diff
@@ -0,0 +1,9 @@
+$box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+$font-family: "Roboto", sans-serif;
+$color-primary: #060b35;
+$color-secondary: #fff;
+$color-border: #ccc;
+$card-height: 300px;
+$card-width: 200px;
+$duration: 300ms;
+$card-gap: 48px;
```

Comment:
Tip: some of these variables (such as `$box-shadow`, `$font-family`, `$color-border`, `$card-height` and `$card-gap`) are used only once, so you don't need those essentially. But, considering its usefulness for controls and readability, most of it can remain. The only variable I would recommend to remove is `$font-family`, it is set to the whole html, only once, and from its position in the code it is pretty obvious where to adjust your settings without a variable. But that's really up to you :)

Reply:
Yeah, agree.